### ISSUE TITLE
Good news from Google.

### ISSUE THREAD

**Issue by efeint01** [SuddenH4X/awesome-app-rating#68]
For In App Reviews:
**Status: Won't Fix (Intended Behavior)**
_We have looked into the issue you have reported and would like to inform that this is working as intended, it is showing only once in prod due to the [quota](https://developer.android.com/guide/playcore/in-app-review#quotas)._

This means that if google play increases your app's quota, it will show more users voting dialog.

And I would like to know if In App Review still works properly in this library.
